mdsuite/calculators/green_kubo_ionic_conductivity.py

```diff
@@ -230,39 +223,80 @@ def _post_operation_processes(self):
         -------
 
         """
-        result = self.prefactor * np.array(self.sigma)
-
+        self.sigmas = np.array(self.sigmas)
+        sigma = np.mean(self.sigmas, axis=0)
+        sigma_SEM = np.std(self.sigmas, axis=0) / np.sqrt(len(self.sigmas))
+
+        self.acfs = np.array(self.acfs)
+        acf = np.mean(self.acfs, axis=0)
+        ionic_conductivity = self.prefactor * sigma[self.args.integration_range - 1]
+        ionic_conductivity_SEM = (
+            self.prefactor * sigma_SEM[self.args.integration_range - 1]
+        )
         data = {
-            self.result_keys[0]: np.mean(result).tolist(),
-            self.result_keys[1]: (np.std(result) / np.sqrt(len(result))).tolist(),
+            self.result_keys[0]: [ionic_conductivity],
+            self.result_keys[1]: [ionic_conductivity_SEM],
```

Comment:
I guess in the long run it would make sense to allow for plain `np.ndarrys` to be saved in data which might not be possible at the moment. 
In this case is there a reason to use `[ionic_conductivity]` instead of `ionic_conductivity.tolist()` ?

Reply:
ionic conductivity here is just a float, if it is converted to a list with the call then no reason. In any case ,I am not sure why this is a list anyway.. it was in previous calculators so I left it. However, a lot of the saving stuff will be updated soon and I'd love to have this PR moved through as it is slowing down API changes from being completed.

Reply:
I guess then it should just be `self.result_keys[0]: ionic_conductivity` because I also don't think it makes sense that it is a list.